Update teams-powershell-release-notes.md (#7694)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-powershell-release-notes.md

| Date | Version | Updates |
|------- | -------------------- | ------------------------------ |

> | August 2021 | [2.5.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.0) |<li>Updates for AccessToken login with Connect-MicrosoftTeams.</li><li>Fixes for Interactive login of Connect-MicrosoftTeams in Cloudshell.</li><li>Improvements to New-Team cmdlet for team creation scenarios.</li><li>TeamsUnassignedNumberTreatment cmdlets are now available.</li><li>Get-CsCsOnlineDialInConferencingBridge and Set-CsOnlineDialInConferencingBridge</li><li>Releases modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only)</li>|

Reviewer: ```suggestion
| August 2021 | [2.5.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.0) |<li>Updates for AccessToken login with Connect-MicrosoftTeams.</li><li>Fixes for Interactive login of Connect-MicrosoftTeams in Cloudshell.</li><li>Improvements to New-Team cmdlet for team creation scenarios.</li><li>TeamsUnassignedNumberTreatment cmdlets are now available.</li><li>Get-CsCsOnlineDialInConferencingBridge and Set-CsOnlineDialInConferencingBridge</li><li>Releases modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only).</li>|
```

> | August 2021 | [2.5.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.0) |<li>Updates for AccessToken login with Connect-MicrosoftTeams.</li><li>Fixes for Interactive login of Connect-MicrosoftTeams in Cloudshell.</li><li>Improvements to New-Team cmdlet for team creation scenarios.</li><li>TeamsUnassignedNumberTreatment cmdlets are now available.</li><li>Get-CsCsOnlineDialInConferencingBridge and Set-CsOnlineDialInConferencingBridge</li><li>Releases modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only)</li>|
> | July 2021 | [2.4.1-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.1-preview) |<li>Grant cmdlets changes now available.</li><li>New Voice related cmdlets are released.</li><li>Removal of certificate thumbprint authentication for -Cs* cmdlets.</li><li>Logging fix for logging files of all cmdlets.</li><li>Fixes issues with *TeamChannelUser cmdlets</li>|
> | June 2021 | [2.4.0-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.0-preview) |<li>Preview only release of modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only), Get-CsOnlineDialInConferencingLanguagesSupported, and Import-CsOnlineAudioFile.</li><li>Modernized versions of Get-CsOnlineDialInConferencingLanguagesSupported and Import-CsOnlineAudioFile are expected to work similar/same to their remoting counter parts.

Reviewer: ```suggestion
| August 2021 | [2.5.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.0) |<li>Updates for AccessToken login with Connect-MicrosoftTeams.</li><li>Fixes for Interactive login of Connect-MicrosoftTeams in Cloudshell.</li><li>Improvements to New-Team cmdlet for team creation scenarios.</li><li>TeamsUnassignedNumberTreatment cmdlets are now available.</li><li>Get-CsCsOnlineDialInConferencingBridge and Set-CsOnlineDialInConferencingBridge.</li><li>Releases modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only).</li>|
| July 2021 | [2.4.1-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.1-preview) |<li>Grant cmdlets changes now available.</li><li>New Voice related cmdlets are released.</li><li>Removal of certificate thumbprint authentication for -Cs* cmdlets.</li><li>Logging fix for logging files of all cmdlets.</li><li>Fixes issues with *TeamChannelUser cmdlets.</li>|
| June 2021 | [2.4.0-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.0-preview) |<li>Preview only release of modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only), Get-CsOnlineDialInConferencingLanguagesSupported, and Import-CsOnlineAudioFile.</li><li>Modernized versions of Get-CsOnlineDialInConferencingLanguagesSupported and Import-CsOnlineAudioFile are expected to work similar/same to their remoting counterparts.
```

>   - Modernized versions of Get-CsTenant and Get-CsOnlineUser (when run with -identity parameter) don't emit deprecated properties.
>   - Modernized versions of Get-CsTenant and Get-CsOnlineUser (when run with -identity parameter) have some formatting changes when compared to their remoting counter parts.</li><li>Releases [Get|Set|Grant|New|Remove]-CsTeamsAudioConferencingPolicy cmdlets.</li><li>Releases Get-CsOnlineAudioFile and Remove-CsOnlineAudioFile cmdlets.</li><li>Set-TeamTargetingHierarchy, Remove-TeamTargetingHierarchy, Get-TeamTargetingHierarchyStatus are now available for GCC customers.</li><li>Fixes the endpoint called by the Get-TeamTargetingHierarchyStatus command.</li>|

Reviewer: ```suggestion
```
